cmake/OpenCVCRTLinkage.cmake

```diff
@@ -77,7 +77,7 @@ else()
   endforeach(flag_var)
 endif()
 
-if(NOT ${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION} LESS 2.8 AND NOT ${CMAKE_MINOR_VERSION}.${CMAKE_PATCH_VERSION} LESS 8.6)
+if(NOT ${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION} LESS 2.8 AND (${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION} GREATER 2.8 OR NOT ${CMAKE_MINOR_VERSION}.${CMAKE_PATCH_VERSION} LESS 8.6))
```

Comment:
`OpenCVMinDepVersions.cmake` defines:

```
set(MIN_VER_CMAKE 2.8.7)
```

so this check can be eliminated.

BTW, compare versions via these expressions (instead of floating point numbers comparison):

```
if(<variable|string> VERSION_LESS <variable|string>)
if(<variable|string> VERSION_EQUAL <variable|string>)
if(<variable|string> VERSION_GREATER <variable|string>)
```

They are available since CMake 2.8.6 (released 3 years ago).